### ISSUE TITLE
Cannot render 9-patch with 0-size tiles.

### ISSUE THREAD

**Issue by lhunath** [vindolin/ninepatch#5]
The following 9-patch has a padding the size of the full image and a stretch area that starts at the bottom and the left.  As a result, the bottom and left tiles are either width and/or height 0.

![divider 9](https://user-images.githubusercontent.com/36948/60541719-3ce8ae00-9ce0-11e9-9469-a46e268ec6da.png)

```
$ ninepatch render "$src" 200 200 out.png
Traceback (most recent call last):
  File "lib/buildtools/ninepatch", line 30, in <module>
    scaled_image = ninepatch.render(args.width, args.height)
  File "/Users/lhunath/Library/Python/2.7/lib/python/site-packages/ninepatch/__init__.py", line 293, in render
    tile = tile.resize((tile.size[0], tile_scale['y'] + extra_y), img_filter)
  File "/Users/lhunath/Library/Python/2.7/lib/python/site-packages/PIL/Image.py", line 1801, in resize
    im = im.resize(size, resample, box)
  File "/Users/lhunath/Library/Python/2.7/lib/python/site-packages/PIL/Image.py", line 1806, in resize
    return self._new(self.im.resize(size, resample, box))
ValueError: height and width must be > 0
```

**Comment by gengyj16** [comment]
line 88: axes = {'x': 0, 'y': 1} in __init__.py -> axes = {'x': 1, 'y': 1}

**Comment by vindolin** [comment]
Totally forgot about that one, I try to look into it in the next days.

**Comment by vindolin** [comment]
> line 88: axes = {'x': 0, 'y': 1} in **init**.py -> axes = {'x': 1, 'y': 1}

That would only result in the y axis being parsed twice, which is certainly not the solution to this problem.

**Comment by vindolin** [comment]
> The following 9-patch has a padding the size of the full image and a stretch area that starts at the bottom and the left. As a result, the bottom and left tiles are either width and/or height 0.
> 
> ![divider 9](https://user-images.githubusercontent.com/36948/60541719-3ce8ae00-9ce0-11e9-9469-a46e268ec6da.png)
> 
> ```
> $ ninepatch render "$src" 200 200 out.png
> Traceback (most recent call last):
>   File "lib/buildtools/ninepatch", line 30, in <module>
>     scaled_image = ninepatch.render(args.width, args.height)
>   File "/Users/lhunath/Library/Python/2.7/lib/python/site-packages/ninepatch/__init__.py", line 293, in render
>     tile = tile.resize((tile.size[0], tile_scale['y'] + extra_y), img_filter)
>   File "/Users/lhunath/Library/Python/2.7/lib/python/site-packages/PIL/Image.py", line 1801, in resize
>     im = im.resize(size, resample, box)
>   File "/Users/lhunath/Library/Python/2.7/lib/python/site-packages/PIL/Image.py", line 1806, in resize
>     return self._new(self.im.resize(size, resample, box))
> ValueError: height and width must be > 0
> ```

**Comment by gengyj16** [comment]
I made a pull request to fix this.
#7 
You can use this image for verification.
![tile_with_zero_size 9](https://github.com/vindolin/ninepatch/assets/22300654/dc6a8cea-9d12-4bd6-be1f-150ed74151b5)

**Comment by vindolin** [comment]
Fixed in 0.2.0 thanks to @gengyj16
